Game/Source/GDKTestGyms/AUptimeGameMode.h

```diff
@@ -2,91 +2,54 @@
 
 #pragma once
 
-#include "BlackboardValues.h"
 #include "CoreMinimal.h"
-#include "BenchmarkGymGameModeBase.h"
-#include "BenchmarkGymNPCSpawner.h"
+#include "BenchmarkGymGameMode.h"
 #include "UptimeCrossServerBeacon.h"
 #include "AUptimeGameMode.generated.h"
 
 DECLARE_LOG_CATEGORY_EXTERN(LogUptimeGymGameMode, Log, All);
 
-typedef TPair<TWeakObjectPtr<AController>, int> ControllerIntegerPair;
-
-/**
- *
- */
 UCLASS()
-class GDKTESTGYMS_API AUptimeGameMode : public ABenchmarkGymGameModeBase
+class GDKTESTGYMS_API AUptimeGameMode : public ABenchmarkGymGameMode
 {
 	GENERATED_BODY()
 public:
 	AUptimeGameMode();
-	AActor* FindPlayerStart_Implementation(AController* Player, const FString& IncomingName) override;
 
-	int32 GetEgressTestSize()const { return TestDataSize; }
-	int32 GetEgressTestNum() const { return TestDataFrequency; }
+	int32 GetEgressTestSize() const { return EgressSize; }
+	int32 GetEgressTestNum() const { return EgressFrequency; }
 
 protected:
-	virtual void BuildExpectedActorCounts() override;
-	virtual void OnAnyWorkerFlagUpdated(const FString& FlagName, const FString& FlagValue) override;
 
 	UPROPERTY(EditAnywhere, NoClear, BlueprintReadOnly, Category = Classes)
 	TSubclassOf<AUptimeCrossServerBeacon> CrossServerClass;
 
-private:
-	TArray<FBlackboardValues> PlayerRunPoints;
-	TArray<FBlackboardValues> NPCRunPoints;
-	void GenerateTestScenarioLocations();
+	virtual void StartCustomNPCSpawning() override;
 
-	TArray<ControllerIntegerPair> AIControlledPlayers;
+	virtual void ReadCommandLineArgs(const FString& CommandLine) override;
+	virtual void ReadWorkerFlagValues(USpatialWorkerFlags* SpatialWorkerFlags) override;
+	virtual void BindWorkerFlagDelegates(USpatialWorkerFlags* SpatialWorkerFlags) override;
+
+private:
 
-	virtual void Tick(float DeltaSeconds) override;
-	virtual void ParsePassedValues() override;
+	int32 EgressSize;
```

Comment:
EgressSize and EgressFrequency don't actually appear to be used atm - are these available for some WIP functionality?

Reply:
AUptimeGymsCharacter pulls the values using getters in the header.